includes/general.php

```diff
@@ -1052,7 +1052,16 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			$return  = '';
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 3ad5a47bde8150a0b6c85eb02e86e30e</sub>